src/mozilla_sec_eia/models/sec10k/sec_output_table.py

```diff
@@ -0,0 +1,367 @@
+"""Module for creating an SEC 10K output table with filing companies and subsidiary companies."""
+
+import logging
+import re
+from importlib import resources
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from dagster import AssetIn, AssetOut, multi_asset
+
+from mozilla_sec_eia.models.sec10k.utils.cloud import (
+    convert_ex21_id_to_filename,
+)
+from mozilla_sec_eia.models.sec_eia_record_linkage.preprocessing import (
+    company_name_cleaner,
+)
+
+from .extract import (
+    sec10k_filing_metadata,
+    year_quarter_partitions,
+)
+
+logger = logging.getLogger(f"catalystcoop.{__name__}")
+
+INVALID_NAMES = [
+    "llc",
+    "limited liability company",
+    "limited",
+    "ltd",
+    "iiii",
+    "inc",
+    "incorporated",
+    "partnership",
+    "i",
+    "name",
+    "company",
+    "&",
+    "",
+]
+
+
+def _remove_weird_sec_cols(sec_df) -> pd.DataFrame:
+    weird_cols = ["]fiscal_year_end", "]irs_number", "]state_of_incorporation"]
```

Comment:
There a couple filings where the field names are messed up, and they turn into weird columns when you pivot the basic 10k table. If we were to do this right, I think we'd remove these fields when they are extracted from the 10k...